src/frequenz/sdk/actor/power_distributing/_component_managers/_battery_manager.py

```diff
@@ -652,40 +652,38 @@ def _parse_result(
 
         for inverter_id, aws in tasks.items():
             battery_ids = self._inv_bats_map[inverter_id]
+            errored = True
```

Comment:
"error" is not a verb, treating it like it is feels really weird.  Other options: `oper_success = False`, `has_error` = True, `failed = True`, `succeeded = False`.

I like `succeeded = False`, `op_successful`, etc., because they seem to presume less.

Reply:
Really? Then it is Internet English, I have seen it used many times. Also Wiktionary has an entry for it: https://en.wiktionary.org/wiki/errored, and some sites even has the conjugation for it: https://www.verbix.com/webverbix/english/error. The Oxford dictionary seems to have an entry for it too: https://www.oed.com/dictionary/error_v?tab=factsheet#5217607, it seems to be used since 1820 :upside_down_face: (but I couldn't find it in the Cambridge dictionary).

I can change it anyways, I was only surprised that it might not be as common as I thought.

Reply:
[Chambers](https://chambers.co.uk/search/?query=error&title=21st) and [Webster](https://www.merriam-webster.com/dictionary/error) also list only the noun form.  OED must have been smoking something.

But `succeeded = False` might be better, because that doesn't imply there was an error, and in context, it just means we don't know yet, unlike `error = True`.

Reply:
I was going to change it but I see you already approved, so I will keep `failed` to avoid the extra review roundtrip as it is such a small detail, I guess it is fine to keep it with `failed`.